### DISCUSSION TITLE
Maybe we can make a nightly neovim?

### DISCUSSION THREAD

**Discussion by jinzhongjia** [msys2/MINGW-packages#18289] [General]
Just like aur, directly download neovim's nightly tag and install it

**Comment by lazka** [comment]
We currently don't have anything like AUR in place. You'd have to maintain your own nightly PKGBUILD file and build/install it from time to time.

We try to avoid having multiple versions of a package in the repo to reduce the maintenance burden.

We are happy to backport fixes though, in case upstream is too slow in creating releases.
